package.json

```diff
@@ -18,12 +18,14 @@
   ],
   "main": "./lib/index.js",
   "scripts": {
+    "gulp": "gulp",
```

Comment:
Do this need to be added with the more common use of npx gulp to run the module from inside the project?